filter-junit.go

```diff
@@ -14,8 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/*
- * This command filters a JUnit file such that only tests with a name
+/*This command filters a JUnit file such that only tests with a name
```

Comment:
This looks really ugly, as pointed out in https://github.com/golang/lint/issues/22.

Please use the form suggested in https://github.com/golang/lint/issues/22#issuecomment-23749508 instead.

Reply:
Thanks! I have updated the PR.

Reply:
@animeshk08 could you update the smb csi driver PR? thanks.